src/Nethermind/Nethermind.Blockchain/BlockHashInState/BlockHashInStateHandler.cs

```diff
@@ -41,8 +40,19 @@ public void AddParentBlockHashToState(BlockHeader blockHeader, IReleaseSpec spec
 
 
         var blockHashWitness = new VerkleExecWitness(NullLogManager.Instance);
-        blockHashWitness.AccessCompleteAccount(eip2935Account);
-        blockHashWitness.AccessForStorage(eip2935Account, blockIndex, true);
+        var gasAvailable = blockHeader.GasLimit - blockHeader.GasUsed;
```

Comment:
there was no gas charge here - we just add witness here, without charging the gas

Reply:
fixed